src/main/java/edu/ucla/library/iiif/auth/TemplateKeys.java

```diff
@@ -0,0 +1,44 @@
+
+package edu.ucla.library.iiif.auth;
+
+/**
+ * Valid keys for Handlebars templates.
+ */
+public final class TemplateKeys {
+
+    /**
+     * The access token object key.
+     */
+    public static final String ACCESS_TOKEN_OBJECT = "access-token-object";
+
+    /**
+     * The campus network key.
+     */
+    public static final String CAMPUS_NETWORK = CookieJsonKeys.CAMPUS_NETWORK;
```

Comment:
Why have a new constant that has same value as CookieJsonKeys.CAMPUS_NETWORK? Why not just use CookieJsonKeys.CAMPUS_NETWORK?

Reply:
The idea is to have separate namespaces for each context that a "key" is used in:
- keys that are used to construct JSON response bodies;
- those used to construct cookies (which are encrypted JSON structures); and
- those used to pass values to Handlebars templates.

The intent is to provide an answer to the question: "which keys are valid for use in rendering Handlebars templates?" (or in constructing cookies, or in constructing JSON-valued HTTP responses).

In this case, the `cookie.hbs` template is used to render an HTML page that provides information to the user about the cookie they've just received. This statement you highlighted is an attempt to communicate that we intend to pass to the Handlebars template the same `CAMPUS_NETWORK` value that was used to construct the cookie. Similarly,

```java
public static final String ORIGIN = Param.ORIGIN;
```

is an attempt to communicate that we intend to pass the Handlebars template the same `ORIGIN` that was provided as a query parameter in the request URL.

It made sense to me and this is my reasoning, but I am open to doing this differently if it's convoluted or confusing. Maybe these classes would make more sense as enums instead?

Reply:
Although, maybe it _is_ a problem that some of these constants reference other constants? E.g., maybe we should be able to change the value of the `TemplateKeys.CAMPUS_NETWORK` without changing the corresponding cookie key...

Reply:
Okay, the latest commit decouples all the IDs that aren't required to have the same value (`ResponseJsonKeys.MESSAGE_ID` is the exception).